--- a/config/config.in	2018-02-15 18:47:21.921771979 +1300
+++ b/config/config.in	2018-02-15 18:47:24.765773157 +1300
@@ -10,7 +10,7 @@
 ###########################################################################
 ## Which speech tools to use
 
-EST=$(TOP)/../speech_tools
+EST?=$(TOP)/../speech_tools
 
 ###########################################################################
 ## Where the festival tree will be installed.
